pkg/ibmvpc/examples/out/explain_out/GroupingExternalSG1_all_vpcs_explain.txt

```diff
@@ -1,4 +1,5 @@
 Explaining connectivity from vsi1-ky to 161.26.0.0/8 within test-vpc1-ky
+Interpreted Src: vsi1-ky[10.240.10.4]
```

Comment:
It seems that indeed it is a bit confusing that there is only `Interpreted Src: vsi1-ky[10.240.10.4]`, especially given that it does not really add information, as here.
I suggest to *always* print both `Interpreted Src:` and `Interpreted Dst:`
 For external:
`Interpreted Dst: 161.26.0.0/8 (external)`

@adisos @haim-kermany what do you think?

Reply:
you are lucky that @zivnevo does not allow me to answer in Hebrew.
 @!#&

Reply:
HaHaHa

Do you want me to take over responsibility for this issue?  
I see no other way then the "trail and error" one

Reply:
fixed